config/initializers/devise.rb

```diff
@@ -257,5 +257,5 @@
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = "/my_engine/users/auth"
 
-  config.omniauth :facebook, "688926831253248", "eba52465b1b4d062aca3d77865413304", scope: "email,user_birthday,read_stream", display: "popup"
+  config.omniauth :facebook, Figaro.env.facebook_api_key, Figaro.env.facebook_secret, scope: "email,user_birthday,read_stream", display: "popup"
```

Comment:
You may want to re-generate these tokens so that if this repo ever gets hacked they don't have access to your facebook secrets :)